plugin/build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     implementation group: 'org.opensearch', name: 'opensearch', version: "${opensearch_version}"
     implementation "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
     // Multi-tenant SDK Client
-    implementation "org.opensearch:opensearch-remote-metadata-sdk:${opensearch_build}"
+    implementation "org.opensearch:opensearch-remote-metadata-sdk:3.0.0-SNAPSHOT"
```

Comment:
Remote sdk should also be `3.0.0.0-alpha1-SNAPSHOT`.

@dbwiddis

Reply:
Same above for this: https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/opensearch-remote-metadata-sdk/

Reply:
> Remote sdk should also be `3.0.0.0-alpha1-SNAPSHOT`.

We are still making changes on that for 2.19.0.   I do not plan on updating to alpha until after at least code freeze date.  

If anyone wants to get ahead of the game they are welcome to check out that repo and publish it locally with alpha1.  All this churn as we're trying to finish up features within the week is causing a lot of wasted effort and delays.

Reply:
@dbwiddis just an update CU main is updated to 3.0.0.0-alpha1-SNAPSHOT:
* https://github.com/opensearch-project/common-utils/pull/775#issuecomment-2616922193

Thanks.